lib/buildkite_test_collector/test_result.ex

```diff
@@ -30,7 +29,7 @@ defmodule BuildkiteTestCollector.TestResult do
 
   @typedoc "Individual test summary.  Spec as yet unconfirmed."
   @type t :: %TestResult{
-          id: UUID.t(),
+          id: String.t(),
```

Comment:
Made it a string, but I can also create a custom type for the UUID if you prefer that.